docs/src/modules/components/Demo.js

```diff
@@ -373,7 +316,7 @@ class Demo extends React.Component {
                     data-ga-event-action="stackblitz"
                     onClick={this.handleClickStackBlitz}
                   >
-                    Edit in StackBlitz
+                    Edit in StackBlitz (JS only)
```

Comment:
I think that we can kill StackBlitz at some point: https://www.similarweb.com/fr/website/codesandbox.io?competitors=stackblitz.com.

Reply:
I agree. It doesn't add that much value to simply add an alternative. Especially now that it has less features. If we could fork existing repos on stackblitz we could make it work since there exists a react-app-ts example. I could only find an option to create new ones from presets though: https://stackblitz.com/docs#stackblitz-api. All of these require a specific main file though.

Reply:
Once codesandbox works on IE11 there won't be a downside